Kajabi/sage-lib
# [SAGE-256] Nav/Nav Link and Sidebar - Foundations Update

## Description
Foundational updates to nav/nav link and sidebar component with the following updates:

- Adjust colors and spacing of text
- Adjust the font size of sub-links
- Adjust colors and spacing focus/active states
- Add a border to the sub-menu section

**NOTE** this PR does not include filled icons or updates to sidebar background colors (the sidebar background will be updated in the Stage Color project). 

## Screenshots

|  Before  |  After  |
|--------|--------|
|![Screen Shot 2022-02-22 at 4 00 49 PM](https://user-images.githubusercontent.com/1175111/155240740-ef93653f-f2cc-40f8-8417-c28663a83aa6.png)|![Screen Shot 2022-02-23 at 11 43 36 AM](https://user-images.githubusercontent.com/1175111/155395987-102d05ee-fba2-49c8-b02b-b7085c20f50b.png)|


## Testing in `sage-lib`

- Navigate to [Sidebar](http://localhost:4000/pages/component/sidebar), [Nav](http://localhost:4000/pages/component/nav), and [Nav Link](http://localhost:4000/pages/component/nav_link)
- Verify updates match foundations [design mock](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%E2%9A%A0%EF%B8%8F-%5BDO-NOT-USE-WIP%5D-Sage-Update?node-id=479%3A45769)


## Testing in `kajabi-products`
1. (**LOW**) Foundational updates to sidebar, nav, and nav link styling to match the new design updates.


## Related
https://kajabi.atlassian.net/browse/SAGE-256
